cloudinit/net/__init__.py

```diff
@@ -839,11 +839,15 @@ def find_entry(mac, driver, device_id):
 
     if len(ops) + len(ups) == 0:
         if len(errors):
-            LOG.debug("unable to do any work for renaming of %s", renames)
+            LOG.warning(
+                "Unable to rename interfaces: %s due to errors: %s",
+                renames,
+                errors,
```

Comment:
Good update, best to always have parametrized error strings to give context. Thanks.